vpr/src/route/route_timing.cpp

```diff
@@ -1281,7 +1369,9 @@ static t_rt_node* setup_routing_resources(int itry,
                                           unsigned num_sinks,
                                           int min_incremental_reroute_fanout,
                                           CBRR& connections_inf,
-                                          t_rt_node** rt_node_of_sink) {
+                                          t_rt_node** rt_node_of_sink,
+                                          const t_router_opts& router_opts,
+                                          bool hold) {
```

Comment:
`hold` seems like a poor variable name.  What is it's purpose?  Can you think of a better variable name?

Reply:
Fair, run_rcv would be a better variable name. The purpose is to allow the router to fully rip up high fanout nets if RCV is enabled, and there is negative hold slack.

Reply:
` ripup_high_fanout_nets` sounds like a significant better name, assuming that it accurately captures the idea behind it